bot/tests/processors/test_pennychat.py

```diff
@@ -32,6 +32,51 @@ def create_penny_chat():
     return chat.id
 
 
+def create_penny_chat__private():
```

Comment:
remove this function and instead create a new `PennyChatSlackInvitationFactory` with all the other factories that defaults to PUBLIC, then when you need this in the test do `penny_chat = PennyChatSlackInvitationFactory(visibility=PennyChat.PUBLIC)` and you can get the id as `penny_chat.id`